src/vs/platform/quickinput/common/quickInput.ts

```diff
@@ -404,6 +405,11 @@ export interface IQuickPick<T extends IQuickPickItem> extends IQuickInput {
 	 */
 	customLabel: string | undefined;
 
+	/**
+	 * The icon for the custom button. Note: do not set customLabel if you set customIcon, otherwise the label formatting will be wrong.
```

Comment:
Not sure why this is or how to fix it, but formatting was wrong when this happened. However, the `Button` object allows us to set both label and icon, so I replicated that behavior here.